### ISSUE TITLE
Form validation: When errors are listed

### ISSUE THREAD

**Issue by juliemercier** [wet-boew/wet-boew#7564]
I am using JAWS to test our applications. To fill out the form I am in forms mode. I notice that when there are errors, focus is indeed placed on the error list at the top of the page but nothing is said by the reader. When I do tab, the first error in the list is read. Would it be possible to put a tabindex="0" on the heading "The form could not be submitted because X errors were found." so that users will understand the general nature of the problem?

**Comment by juliemercier** [comment]
To clarify: This is for IE 11 and JAWS 15. When I use NVDA, it does say "The form could not be submitted because X errors were found." So I guess this is a JAWS technology issue. It simply reads "Region".

**Comment by hsrudnicki** [comment]
Does this remain still an issue? Please update the status of this again, whether this can be closed or otherwise? Thx

**Comment by kodecount** [comment]
Yes this is a Jaws issue it declares alerts as a region. I say this can be closed as it not really issue with WET.
